compiler/plugins/target/LLVMCPU/LLVMCPUTarget.cpp

```diff
@@ -515,6 +517,24 @@ class LLVMCPUTargetBackend final : public TargetBackend {
                         variantOp.getName(), ".codegen.bc", *llvmModule);
     }
 
+    {
+      const char* codegenModuleFileEnv = getenv("CODEGEN_MODULE_FILE");
```

Comment:
what exactly is this for? we really don't want to be adding env vars if we can avoid it